### DISCUSSION TITLE
Provide function name mapping to alive-tv

### DISCUSSION THREAD

**Discussion by momvart** [AliveToolkit/alive2#1106] [Q&A] [answered]
Let's say I want to run Alive on two `ll` files with function names suffixed differently, e.g., `f` appears as `f_xyz` in the first file and as `f_abc` in the second one. By default, `alive-tv` ignores `f` because the name doesn't match.

Is there a way to tell how functions should be mapped between the two files?

**Comment by nunoplopes** [comment, marked as answer]
alive-tv doesn't support that use case currently. But if you concatenate the two files, you can use `-src-fn=source -tgt-fn=target`.

**Reply by momvart** [reply]
Thanks.
I think it is valuable to add at some point. For example, Rust mangles function names with a hash, which might become different given two versions.
